stacker/lookups/handlers/rxref.py

```diff
@@ -11,13 +11,31 @@
         some-relative-fully-qualified-stack-name::SomeOutputName}
 
 """
-from functools import partial
-
-from .output import handler as output_handler
+from .output import deconstruct
 
 TYPE_NAME = "rxref"
 
-# rxref is the same as the `output` handler, except the value already contains
-# the relative fully qualified name for the stack we're fetching the output
-# from.
-handler = partial(output_handler, rfqn=True)
+
+def handler(value, provider=None, context=None, **kwargs):
```

Comment:
Since the `xref` and `rxref` lookups can't re-use the `output` lookup, I just went ahead and re-implemented them to be more explicit, since they were pretty hard to follow before with all of the function currying.